iclr2017/python/assign-reviewers.py

```diff
@@ -1,145 +0,0 @@
-#!/usr/bin/python
```

Comment:
I don't know the best way to do this with git - I'm not sure if Melisa knows, but I'm taking this opportunity to look it up